l10n_it_fatturapa_pec/models/mail_thread.py

```diff
@@ -178,7 +179,7 @@ def find_attachment_by_subject(self, subject):
                 return fatturapa_attachment_out
         return False
 
-    def create_fatturapa_attachment_in(self, attachment):
+    def create_fatturapa_attachment_in(self, attachment, message_dict):
```

Comment:
@robyf70 per non rischiare di rompere eventuali moduli che fanno override di `create_fatturapa_attachment_in`, metterei
`def create_fatturapa_attachment_in(self, attachment, message_dict=None):`
e poi dopo
`if message_dict is not None 'date' in message_dict:`

Reply:
@eLBati Fatto